sirepo/pkcli/raydata.py

```diff
@@ -4,9 +4,147 @@
 :copyright: Copyright (c) 2021 RadiaSoft LLC.  All Rights Reserved.
 :license: http://www.apache.org/licenses/LICENSE-2.0.html
 """
+from pykern import pkasyncio
+from pykern import pkcompat
+from pykern import pkconfig
+from pykern import pkio
+from pykern import pkjson
 from pykern.pkcollections import PKDict
-from pykern.pkdebug import pkdp, pkdlog
+from pykern.pkdebug import pkdp, pkdlog, pkdexc, pkdformat
 from sirepo.template import template_common
+import aenum
+import asyncio
+import base64
+import copy
+import databroker
+import databroker.queries
+import event_model
+import io
+import random
+import requests
+import sirepo.feature_config
+import sirepo.srtime
+import sqlalchemy
+import sqlalchemy.ext.declarative
+import sqlalchemy.orm
+import time
+import tornado.websocket
+import uuid
+import zipfile
+
```

Comment:
I removed an import from replay that seemed unnecessary  https://github.com/radiasoft/raydata/blob/c7b02306b9111ac2584ebb256a3a3615361494d5/raydata/pkcli/replay.py#L18